apps/landingPage/src/components/Footer.js

```diff
@@ -71,6 +72,11 @@ const FooterContainer = styled.div`
   justify-content: space-between;
   height: 105px;
   align-items: center;
+  @media (max-width: ${BREAKPOINTS.BIG_MOBILE}px) {
```

Comment:
@RobinCsl should we follow `mobile-first` here as well?

Reply:
I believe we should follow `mobile-first` approach everywhere. Thanks for pointing it out!

Reply:
Let's just get this merged, and we can create a new PR to refactor media-queries to mobile first